src/component/footer/Footer.jsx

```diff
@@ -1,83 +1,135 @@
+                  </div>
+                </div>
+              </div>
+              <div className="col-lg-6">
+                <div className="footer-right" data-black-overlay="6">
+                  <div className="row">
+                    {/* Start Single Widget  */}
+                    <div className="col-lg-6 col-sm-6 col-12">
+                      <div className="footer-link">
+                        <h4>Quick Links</h4>
+                        <ul className="ft-link">
+                          <li>
+                            <a
+                              href="https://codesydney.pythonanywhere.com/"
+                              target="_blank"
+                            >
+                              Legacy Website
+                            </a>
+                          </li>
+                        </ul>
+                      </div>
+                    </div>
+                    {/* End Single Widget  */}
+                    {/* Start Single Widget  */}
+                    <div className="col-lg-6 col-sm-6 col-12 mt_mobile--30">
+                      <div className="footer-link">
+                        <h4>Say Hello</h4>
+                        <ul className="ft-link">
+                          <li>
+                            <a href="mailto:info@code.sydney">
+                              info@code.sydney
+                            </a>
+                          </li>
+                        </ul>
 
-                                        <div className="col-lg-12">
-                                            <div className="copyright-text">
-                                                <p>Copyright © 2022 Code.Sydney Pty Ltd ABN 37 625 436 151</p>
-                                            </div>
-                                        </div>
-                                    </div>
-                                </div>
-                            </div>
+                        <div className="social-share-inner">
+                          <ul className="social-share social-style--2 d-flex justify-content-start liststyle mt--15">
+                            {SocialShare.map((val, i) => (
+                              <li key={i}>
+                                <a href={`${val.link}`}>{val.Social}</a>
+                              </li>
+                            ))}
+                          </ul>
                         </div>
+                      </div>
+                    </div>
+                    {/* End Single Widget  */}
+
+                    <div className="col-lg-12">
+                      <div className="acknowledge-country">
+                        <br />
+                        <p>
+                          We acknowledge and pay respects to the First Nations
+                          People of Australia, the land in which we live and
+                          work. we recognise the strength, resilience and
```

Comment:
Can you please capitalise the first letter of the the second sentence, i.e. "We recognise"

Reply:
Will do